app/models/equipment_item.rb

```diff
@@ -71,37 +71,34 @@ def make_switch_notes(old_res, new_res, handler)
     old_res_msg = old_res ? old_res.md_link : 'available'
     new_res_msg = new_res ? new_res.md_link : 'available'
     update_attributes(notes: "#### Switched by #{handler.md_link} from "\
-      "#{old_res_msg} to #{new_res_msg} on "\
-      "#{Time.zone.now.to_s(:long)}\n\n" + notes)
+                      "#{old_res_msg} to #{new_res_msg} on "\
+                      "#{Time.zone.now.to_s(:long)}\n\n" + notes)
   end
 
   def update(current_user, new_params) # rubocop:disable all
     assign_attributes(new_params)
     changes = self.changes
-    if changes.empty?
-      return self
-    else
-      new_notes = "#### Edited at #{Time.zone.now.to_s(:long)} by "\
-        "#{current_user.md_link}\n\n"
-      new_notes += "\n\n#### Changes:"
-      changes.each do |param, diff|
-        case param
-        when 'name'
-          name = 'Name'
-          old_val = diff[0].to_s
-          new_val = diff[1].to_s
-        when 'serial'
-          name = 'Serial'
-          old_val = diff[0].to_s
-          new_val = diff[1].to_s
-        when 'equipment_model_id'
-          name = 'Equipment Model'
-          old_val = diff[0] ? EquipmentModel.find(diff[0]).name : 'nil'
-          new_val = diff[1] ? EquipmentModel.find(diff[1]).name : 'nil'
-        end
-        new_notes += "\n#{name} changed from " + old_val + ' to ' + new_val\
-          + '.' if old_val && new_val
+    return self if changes.empty?
+    new_notes = "#### Edited at #{Time.zone.now.to_s(:long)} by "\
+      "#{current_user.md_link}\n\n"
+    new_notes += "\n\n#### Changes:"
+    changes.each do |param, diff|
+      case param
+      when 'name'
+        name = 'Name'
+        old_val = diff[0].to_s
+        new_val = diff[1].to_s
+      when 'serial'
+        name = 'Serial'
+        old_val = diff[0].to_s
+        new_val = diff[1].to_s
+      when 'equipment_model_id'
+        name = 'Equipment Model'
+        old_val = diff[0] ? EquipmentModel.find(diff[0]).name : 'nil'
+        new_val = diff[1] ? EquipmentModel.find(diff[1]).name : 'nil'
       end
+      new_notes += "\n#{name} changed from " + old_val + ' to ' + new_val\
+        + '.' if old_val && new_val
```

Comment:
same as [this](https://github.com/YaleSTC/reservations/pull/1565/files#r64673683)